### PR TITLE
Added Advanced Options To the authoring Page

### PR DESCRIPTION
**Pull request by Curious72**
screenshot from 2016-02-07 04-50-22
screenshot from 2016-02-07 04-50-34

I tested this on Mozilla firefox which doesnt support datetime-local field , couldnt install google-chrome on my ubuntu system , so i am not sure if the drop down datetimelocal component would show up in google-chrome!

Pull request for #646

### CODE REVIEW

File: static/templates/project/authoring.html [hunk 1]
```diff
@@ -79,23 +78,46 @@
...
+                    <md-icon ng-click="project.removeFile(file.id)" class="_file-action"
+                                         md-font-set="material-icons">clear
+                    </md-icon>
+                    </md-list-item>
+                    </md-list>     
+                </div>       
+                <div style="height:auto;text-align:right;width: 900px">
+                  <md-button ng-click="project.isClicked()" style="font-size:10px;">
+                    Advanced
+                    <span ng-show="!project.isExpanded"> ►
```
Comment (dmorina) [inline static/templates/project/authoring.html]: use material icons > and \/

File: static/templates/project/authoring.html [hunk 2]
```diff
@@ -79,23 +78,46 @@
...
+                    <div class="md-list-item-text">
+                        <div><a class="_file-upload">{{ file.name }}</a>{{ ' ('+file.size+')'}}</div>
+                    </div>
+                    <md-icon ng-click="project.removeFile(file.id)" class="_file-action"
+                                         md-font-set="material-icons">clear
+                    </md-icon>
+                    </md-list-item>
+                    </md-list>     
+                </div>       
+                <div style="height:auto;text-align:right;width: 900px">
```
Comment (dmorina) [inline static/templates/project/authoring.html]: no static width should grow within the parent div

**Comment by Curious72** [comment]
@dmorina  @shirishgoyal Please review the PR

**Comment by shirishgoyal** [comment]
PR can be closed as #663 extends it further.

**Pull request closed without merging**